Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOB and DOV cannot be future dated - unauthenticated user", () => {
+        const d = new Date();
+        const year = d.getFullYear();
+        const monthNumber = d.getMonth(); //Maps to monthNames constant array. Index starts at 0
+        const month = d.getMonth() + 1; //0 is January but UI index starts at 1 for January
+        const nextMonth = d.getMonth() === 11 ? 1 : month + 1; //When current month is December, next month will be January
+        const day = d.getDate();
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        // Future Date
+        d.setDate(d.getDate() + 1);
+        const nextYear = d.getFullYear() === year ? year + 1 : d.getFullYear();
+        const nextDay = d.getDate();
+
+        // Test Future Year does not exist
+        selectNotExist(dobYearSelector, nextYear.toString());
+        selectNotExist(dovYearSelector, nextYear.toString());
+
+        // Test Current Year
+        select(dobYearSelector, year.toString());
+        select(dovYearSelector, year.toString());
+
+        if (nextMonth > 1) {
+            // Current year has been set in dropdown, so if next month is 1 - January, it means
+            // current month is December.
+            // Test Future Month does not exist. Month can only be current or past month for current year.
+            selectNotExist(dobMonthSelector, nextMonth);
+            selectNotExist(dovMonthSelector, nextMonth);
+        }
+
+        // Test and set Current Month
+        select(dobMonthSelector, monthNames[monthNumber]);
+        select(dovMonthSelector, monthNames[monthNumber]);
+
+        if (nextDay > 1) {
+            // Current Year and Month have been set. If next day is 1, it means previous.
+            // day was last day of current month. Next Day is associated with the current month.
+            // Test Future Day in current month does not exist.
+            selectNotExist(dobDaySelector, nextDay);
+            selectNotExist(dovDaySelector, nextDay);
+        }
+        //Test Current Day exists
+        selectExist(dobDaySelector, day);
+        selectExist(dovDaySelector, day);
+    });
+
+    it("Vaccination Card - DOB Year and DOV not entered via Click Enter - unauthenticated user", () => {
+        const dobMonth = "June";
+        const dobDay = "15";
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dobMonthSelector, dobMonth);
+
+        select(dobDaySelector, dobDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
```

Comment:
Same note as above - I don't think we need to check date of vaccination in this test since it's focused on birthdate.  Applies to the following tests as well.

Reply:
I want to make sure all messages get displayed when DOB and DOV validation errors occur at the same time.